package.json

```diff
@@ -29,73 +29,65 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
-    "bunsen-core": "0.18.1",
     "chai-jquery": "^2.0.0",
+    "ember-ajax": "2.5.3",
     "ember-async-image": "0.1.1",
-    "ember-block-slots": "^1.1.0",
     "ember-browserify": "^1.1.13",
-    "ember-bunsen-core": "0.12.1",
     "ember-cli": "~2.8.0",
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-autoprefixer": "0.6.0",
     "ember-cli-babel": "^5.1.5",
-    "ember-cli-chai": "0.3.2",
+    "ember-cli-chai": "^0.3.2",
     "ember-cli-code-coverage": "0.3.5",
     "ember-cli-content-security-policy": "^0.5.0",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-ic-ajax": "^0.2.4",
     "ember-cli-import-polyfill": "0.2.0",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mocha": "0.13.1",
-    "ember-cli-moment-shim": "~2.2.0",
+    "ember-cli-mocha": "^0.13.2",
+    "ember-cli-moment-shim": "^3.0.1",
     "ember-cli-notifications": "^4.1.5",
     "ember-cli-uglify": "^1.2.0",
     "ember-code-snippet": "1.8.0",
-    "ember-computed-decorators": "0.3.0",
-    "ember-concurrency": "0.7.17",
+    "ember-computed-decorators": "^0.2.2",
+    "ember-concurrency": "^0.7.19",
```

Comment:
also agree. I think I got these when I updated one of the packages via ember g. I'll go through this file again.